lib/3scale/dotenv.rb

```diff
@@ -0,0 +1,20 @@
+# Ensure the proper .env file is loaded
+#
+# Don't load any .env file for production
+# First, try to load `.env.#{ENV['RACK_ENV']}`
+# If doesn't exist, try to load .env.test
+# If doesn't exist, try to load .env
+def env_file
+  file = ".env.#{ENV['RACK_ENV']}"
+  return file if File.exists?(file)
+
+  File.exists?('.env.test') ? '.env.test' : '.env'
+end
+
+begin
+  return if ENV['RACK_ENV'] == 'production'
```

Comment:
I noticed [this thread](https://github.com/3scale/apisonator/pull/373#discussion_r1516708286), but will reply here, because it's relevant to this line.

@akostadinov raised a good point, indeed if we use `RACK_ENV=staging` in staging, this part will actually fail, because it will try to require `dotenv`, but it's not available in staging.

On the other hand, if we start using `RACK_ENV=production` in staging (which I am not opposed to), then we need to add a new environment variable to set the Bugsnag release stage, because currently we are using the value from `RACK_ENV`.

Reply:
Too late!! Now it'll never be fixed!